util/const.go

```diff
@@ -41,8 +41,8 @@ const (
 // remote
 const (
 	// openGemini-UP
-	Remote_bin_rel_path = "bin/"
-	Remote_etc_rel_path = "etc/"
+	Remote_bin_rel_path = "/bin/"
```

Comment:
Use `filepath.Join`

This way you won’t have to worry about `/` whether it’s too much or too little.

Reply:
I have fixed it.